### PR TITLE
ubam post: add images

### PR DESCRIPTION
**Pull request by lskatz**
Added two images and added an image to the front matter

### CODE REVIEW

File: _posts/2024-06-06-switch-to-bam.md [hunk 1]
```diff
@@ -13,10 +14,13 @@ contributors:
 reviewers:
   - pmenzel
   - hexylena
+last_modified_at: 2024-06-07
 ---
 
 uBAM is a strictly better storage format for your sequencing data, and you should switch to it today.
 
+{% link assets/images/ubam-gather-round %}
```
Comment (hexylena) [inline _posts/2024-06-06-switch-to-bam.md:22]: missing the file extension

File: _posts/2024-06-06-switch-to-bam.md [hunk 2]
```diff
@@ -13,10 +14,13 @@ contributors:
 reviewers:
   - pmenzel
   - hexylena
+last_modified_at: 2024-06-07
```
Comment (hexylena) [inline _posts/2024-06-06-switch-to-bam.md:17]: shouldn't be necessary, we calculate this from git now :) we have the latest commit to identify last updated.

**Review by hexylena — COMMENTED** [review]
good additions!

File: _posts/2024-06-06-switch-to-bam.md (hunk 1 above)
Comment (lskatz) [reply _posts/2024-06-06-switch-to-bam.md]: I fixed the extension and renamed the file

File: _posts/2024-06-06-switch-to-bam.md (hunk 2 above)
Comment (lskatz) [reply _posts/2024-06-06-switch-to-bam.md]: I removed this front matter

**Review by hexylena — APPROVED** [review]

**Pull request merged**